bcda/router.go

```diff
@@ -29,6 +29,7 @@ func NewRouter() http.Handler {
 	r.Route("/api/v1", func(r chi.Router) {
 		r.With(auth.RequireTokenAuth).With(ValidateBulkRequestHeaders).Get("/Patient/$export", bulkRequest)
 		r.With(auth.RequireTokenAuth).Get("/jobs/{jobId}", jobStatus)
+		r.Get("/metadata", metadata)
```

Comment:
Probably a question for @rnagle ... is it OK to expose this endpoint without requiring a token to access?